napari/components/dims.py

```diff
@@ -21,11 +22,10 @@ class Dims(EventedModel):
     last_used : int
         Dimension which was last used.
     range : tuple of 3-tuple of float
-        List of tuples (min, max, step), one for each dimension. In a world
-        coordinates space. As with Python's `range` and `slice`, max is not
-        included.
-    current_step : tuple of int
-        Tuple of the slider position for each dims slider, in slider coordinates.
+        List of tuples (min, max, step), one for each dimension in world
+        coordinates space.
+    span : tuple of 3-tuple of float
```

Comment:
```suggestion
    span : tuple of 2-tuple of float
```

Or `pair` instead of `2-tuple`.